doc/releases/reef.rst

```diff
@@ -0,0 +1,266 @@
+
+      * Node-exporter 1.5.0
+
+      * Grafana 9.4.7
+
+      * Alertmanager 0.25.0
+
+MGR
+~~~
+
+* mgr/snap_schedule: The snap-schedule manager module now retains one snapshot
+  less than the number mentioned against the config option
+  ``mds_max_snaps_per_dir``. This means that a new snapshot can be created and
+  retained during the next schedule run.
+* The ``ceph mgr dump`` command now outputs ``last_failure_osd_epoch`` and
+  ``active_clients`` fields at the top level. Previously, these fields were
+  output under the ``always_on_modules`` field.
+  
+RADOS
+~~~~~
+
+* Trimming of PGLog dups is now controlled by the size rather than the version.
+  This change fixes the PGLog inflation issue that was happening when the
+  online trimming got jammed after a PG split operation. Also, a new offline
+  mechanism has been added: ``ceph-objectstore-tool`` has a new operation
+  called ``trim-pg-log-dups`` that targets situations in which an OSD is unable
+  to boot because of the inflated dups. In such situations, the "You can be hit
+  by THE DUPS BUG" warning is visible in OSD logs. Relevant tracker:
+  https://tracker.ceph.com/issues/53729
+* The RADOS Python bindings are now able to process (opt-in) omap keys as bytes
+  objects. This allows interacting with RADOS omap keys that are not
+  decodeable as UTF-8 strings.
+* mClock Scheduler: The mClock scheduler (the default scheduler in Quincy) has
+  undergone significant usability and design improvements to address the slow
+  backfill issue. The following is a list of some important changes:
+
+  * The ``balanced`` profile is set as the default mClock profile as a
+    compromise between prioritizing client I/O and prioritizing recovery I/O.
+    Users can then choose either the ``high_client_ops`` profile to prioritize
+    client I/O or the ``high_recovery_ops`` profile to prioritize recovery I/O.
+  * QoS parameters including ``reservation`` and ``limit`` are now specified in 
+    terms of a fraction (range: 0.0 to 1.0) of the OSD's IOPS capacity.
+  * The cost parameters (``osd_mclock_cost_per_io_usec_*`` and
+    ``osd_mclock_cost_per_byte_usec_*``) have been removed. The cost of an 
+    operation is now a function of the random IOPS and maximum sequential
+    bandwidth capability of the OSD's underlying device.
+  * Degraded object recovery is given higher priority than misplaced
+    object recovery because degraded objects present a data safety issue that
+    is not present with objects that are merely misplaced. As a result,
+    backfilling operations with the ``balanced`` and ``high_client_ops`` mClock
+    profiles might progress more slowly than in the past, when backfilling
+    operations used the 'WeightedPriorityQueue' (WPQ) scheduler.
+  * The QoS allocations in all the mClock profiles are optimized in 
+    accordance with the above fixes and enhancements.
+  * For more details, see:
+    https://docs.ceph.com/en/latest/rados/configuration/mclock-config-ref/
```

Comment:
reef release notes should probably point at the `reef` version of the docs instead of `latest`

Reply:
I've made this change.